project-api/ballerina-projects/src/main/java/io/ballerina/projects/Package.java

```diff
@@ -63,4 +64,8 @@ public Module getDefaultModule() {
     public BallerinaToml ballerinaToml() {
         return this.packageContext.ballerinaToml();
     }
+
+    public Path packagePath() {
```

Comment:
Looks like this method is unused. If so, can we remove it?

Reply:
It is used inside `write` in `BaloWriter` class